redbot/core/bank.py

```diff
@@ -332,6 +332,26 @@ def _invalid_amount(amount: int) -> bool:
         await _conf.clear_all_members(guild)
 
 
+async def bank_local_clean(guild: discord.Guild, member: discord.member = None):
+    """Remove data for members who are no longer in the server"""
+    # FIXME: use-config-bulk-update
+    # We do not recommend that devs use this protected method as it is not safe to be used in
+    # all cased and this can end up corrupting your config file
```

Comment:
This is a run on and has poor grammar.

Reply:
Think this is improved now let me know if you disagree and suggestions around it